tests/sched/sched_test.go

```diff
@@ -0,0 +1,91 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
you also need to update https://github.com/portworx/torpedo/blob/master/deployments/deploy-ssh.sh#L111 with the .test file for this new test.